### PR TITLE
OSDOCS2582: include overview for the backup and restore book

### PR DESCRIPTION
**Pull request by chinmayi-chandrasekar**
Applies to 4.6+

https://issues.redhat.com/browse/OSDOCS-2582

[Docs preview](https://deploy-preview-37905--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/index.html)

Requires QE approval from @geliu2016

### CODE REVIEW

**Review by michaelryanpeter — COMMENTED** [review]
@chinmayi-chandrasekar 

I have made some suggestions for organizing and perhaps rephrasing some of the content. I think you have written a good overview of the backup and restore process. I think a little reorganization will go a long way for scanability and really make this shine.

Please do what you will with the suggestions, and I hope it helps. Another member of the peer review squad will follow up to check my review.

**Review by geliu2016 — APPROVED** [review]
/lgtm, and I have a question is this overview be added in backup and restore section or any other place? https://docs.openshift.com/container-platform/4.9/backup_and_restore/backing-up-etcd.html

File: _topic_map.yml [hunk 1]
```diff
@@ -2127,6 +2127,8 @@ Name: Backup and restore
 Dir: backup_and_restore
 Distros: openshift-origin,openshift-enterprise
 Topics:
+- Name: Overview of nodes
```
Comment (kalexand-rh) [inline _topic_map.yml:2130]: Please confirm that the title here is what you expect.

Comment (chinmayi-chandrasekar) [reply _topic_map.yml]: The title looks wrong; have changed it now.

**Pull request merged**